service/history/ndc_standby_task_util.go

```diff
@@ -136,14 +137,14 @@ func newHistoryResendInfo(
 func newActivityTaskPostActionInfo(
 	mutableState workflow.MutableState,
 	activityScheduleToStartTimeout time.Duration,
-	useCompatibleVersion bool,
+	activityInfo *persistencespb.ActivityInfo,
```

Comment:
nit: looks like we only need activityInfo here. schedule to start timeout get be get from the activityInfo?